spec/services/doc_auth/lexis_nexis/responses/true_id_response_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe DocAuth::LexisNexis::Responses::TrueIdResponse do
-  let(:success_response_body) { LexisNexisFixtures.true_id_response_success_2 }
+  let(:success_response_body) { LexisNexisFixtures.true_id_response_success_3 }
```

Comment:
It looks like with this change, `true_id_response_success_2` is no longer used. Should we delete it (or rename `true_id_response_success_3` to `_2` and replace the content)?

Reply:
@night-jellyfish , good to notice, i kept it there intend for a reference there so people can see changes. Anyway we can delete it if desired.

Reply:
@night-jellyfish , removed.

Reply:
keeping the 3 in name, since we still have the true_id_response_success.json, this makes me think it was intended to keep a history of the test files.

Reply:
Thank you!